meta-dts-distro/recipes-dts/dts/dts/dts-functions.sh

```diff
@@ -391,7 +391,7 @@ board_config() {
               NEED_SMMSTORE_MIGRATION="true"
               NEED_BOOTSPLASH_MIGRATION="false"
               NEED_BLOB_TRANSMISSION="false"
-              PROGRAMMER_BIOS="internal"
+              PROGRAMMER_BIOS="internal:boardmismatch=force"
```

Comment:
@miczyg1 is `boardmismatch` flag needed when we flash Dasharo firmware?

Reply:
We introduced the board mismatch with Dasharo because v0.9.0 has this lacking space between DDR4 and (MS-7E06) @TomaszAIR